apps/src/javalab/redux/consoleRedux.ts

```diff
@@ -0,0 +1,61 @@
+// Redux store for state that relates to the Java Lab console.
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+
+interface ConsoleLog {
+  type: string;
```

Comment:
should we go ahead and specify this as a union type too? (`'input' | 'output' | 'newline' | 'markdown'`)

Reply:
oh good catch!